sklearn/utils/setup.py

```diff
@@ -58,6 +57,7 @@ def configuration(parent_package='', top_path=None):
 
         with open(pyxfiles, "r") as f:
             tmpl = f.read()
+        from Cython import Tempita  # noqa
```

Comment:
Could this be a function in `sklearn/_build_utils/__init__.py`. Something along these lines:

```py
def tempita_sub(*args, **kwargs):
    """Helper function.
    This does lazy import of Cython to make sure that we do not depend on Cython when building from a  release sdist (since it contains the .c files already)
    """
    from Cython import Tempita
    Tempita.sub(*args, **kwargs)
```

Reply:
Or adding a comment above the import about it.